frontend/src/components/ProfileHistory/ProfileHistory.jsx

```diff
@@ -39,7 +39,7 @@ export default function ProfileHistory({ username }) {
     return (
         <div className="profile-history">
```

Comment:
another nice-to-have would be to see what the user logged in (the one viewing the page), and the viewed user (the one whose profile is being viewed) have in common - Ex. what stocks they both like, how many stocks do they both follow, etc.

Reply:
Will do, I'm gonna remake the profile page soon.